tracee-ebpf/tracee/tracee.go

```diff
@@ -1108,6 +1151,35 @@ func (t *Tracee) processEvent(ctx *context, args map[argTag]interface{}) error {
 	return nil
 }
 
+func getFileHash(fileName string) string {
+	f, _ := os.Open(fileName)
+	if f != nil {
+		defer f.Close()
+		h := sha256.New()
+		_, _ = io.Copy(h, f)
+		return hex.EncodeToString(h.Sum(nil))
+	}
+	return ""
+}
+
+func (t *Tracee) updateProfile(ctx *context, sourceFilePath string, sourceFileCtime int64) {
+	sourceFileSHA := getFileHash(sourceFilePath)
+	if pf, ok := t.profiledFiles[sourceFilePath]; !ok {
+		t.profiledFiles[sourceFilePath] = profilerInfo{
+			MountNS:   ctx.MntID,
+			TimeStamp: sourceFileCtime,
+			Times:     1,
+			FileHash:  sourceFileSHA,
```

Comment:
If you agree and apply the rest of my comments, then the profilerInfo struct is unneeded. The map value can be just "times" (int)

Reply:
What about the file hash?

Reply:
see https://github.com/aquasecurity/tracee/pull/725/files#r636341189

Reply:
I'm not sure if I follow. Are you suggesting we remove the SHA altogether from the profile thats saved to disk?

Reply:
As discussed offline https://github.com/aquasecurity/tracee/pull/725/commits/93f7ce2aae884a80a52aad0590387e9042770676